CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,30 @@
 
 import Foundation
 
-print("Hello, World!")
+func print_icecream (width: Int, height: Int) {
+    for _ in 1...height {           // flowchart - h
+        for _ in 1...width {        // flowchart - w
+            print("* ", terminator: "")
+        }
+        print("\n")
+    }
+}
 
+func print_bar (width: Int) {
+    for _ in 1...4 {                // flowchart - stick_h
+        for w in 1...width {
+            if (w == (width/2) || w == (width/2+2)) {
+                print("| ", terminator: "")
+            }
+            else {
```

Comment:
if else문을 사용할 경우, 일반적으로 스위프트에서는 개행을 하지 않습니다!
```suggestion
            } else {
```

Reply:
🙀 감사합니다!!! 👍 또 새로운 것을 배웠네요!!